src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -229,12 +229,12 @@ public void unsubscribe() {
       } catch (OffsetOutOfRangeException | NoOffsetForPartitionException oe) {
         switch (_offsetResetStrategy) {
           case EARLIEST:
-            _kafkaConsumer.seekToBeginning(oe.partitions());
             oe.partitions().forEach(_consumerRecordsProcessor::clear);
```

Comment:
If EARLIEST or LATEST is in the interval of offsets tracked by the consumer records processor then should this clear the state of the consumer records processor?